helm/values.yaml

```
@@ -12,6 +12,11 @@ cpfs:
 # operatorNamespace: ""
 # servicesNamespace: ""
 
+global:
+  tetheredNamespace: [tenant1,tenant2]
```

```suggestion
  tetheredNamespace: []
```

leave it empty by default